redi-connect-front/src/components/organisms/SideMenu.scss

```diff
@@ -1,11 +1,21 @@
+@import "~bulma/sass/utilities/_all";
 @import "../../styles/_variables.scss";
 
 .side-menu {
-  &__item {
+  @include mobile() {
+    display: flex;
+    justify-content: space-around;
+  }
 
+  &__item {
     &:not(:last-child) {
       margin-bottom: 1rem;
```

Comment:
Would wrap the margin 1 with tablet. That way you can delete the margin 0 so you don't need to overwrite yourself

Reply:
done

Reply:
done